tests/config/conf_wrong_data_and_sampling.yml

```diff
@@ -38,18 +38,18 @@ LBAF_Viz:
   rank_qoi: None
   object_qoi: None
 
-# Simulate from samplers (comment out if reading from log)
+# Other
 from_samplers:
   n_objects: 200
   n_mapped_ranks: 2
   communication_degree: 20
   load_sampler:
     name: lognormal
     parameters:
-      - 1.0
-      - 10.0
+    - 1.0
```

Comment:
Why did you remove the indentation here?

Reply:
pyyaml does it but as written previously we will fix it using a workaround